File: spec/python/test_enum_for_unknown_id.py
import unittest

from enum_for_unknown_id import EnumForUnknownId

class TestEnumForUnknownId(unittest.TestCase):
    def test_enum_for_unknown_id(self):
        with EnumForUnknownId.from_file("src/fixed_struct.bin") as r:
            self.assertEqual(r.one, 80)

File: spec/lua/test_enum_for_unknown_id.lua
local luaunit = require("luaunit")

require("enum_for_unknown_id")

TestEnumForUnknownId = {}

function TestEnumForUnknownId:test_enum_for_unknown_id()
    local r = EnumForUnknownId:from_file("src/fixed_struct.bin")

    luaunit.assertNil(r.one)
end
